PeerioTechnologies/peerio-desktop
# feat: don't display insecure images

#### Relevant info and issue/PR links 

https://app.clubhouse.io/peerio/story/15515/desktop-notice-for-insecure-image-preview

#### Testing instructions  

Should be tested with this icebear PR: https://github.com/PeerioTechnologies/peerio-icebear/pull/326

1. Message with some insecure link (e.g. `http://beebom-redkapmedia.netdna-ssl.com/wp-content/uploads/2016/01/Reverse-Image-Search-Engines-Apps-And-Its-Uses-2016.jpg`)

2. Check that "Image is not displayed..." is shown.


----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
